src/main/java/org/jboss/pnc/api/reqour/dto/RepositoryCloneRequest.java

```diff
@@ -37,10 +38,10 @@
 public class RepositoryCloneRequest {
 
     /**
-     * Type of scm: usually: git
+     * Type of scm
      */
-    @NotBlank
-    String type;
+    @NotNull
+    SCMType type;
```

Comment:
Is this used?

Reply:
discussed in slack, agreed not to delete this